spec/controllers/rooms_controller_spec.rb

```diff
@@ -90,4 +90,25 @@
       expect(response).to have_http_status(:created)
     end
   end
+
+  describe '#recordings' do
+    it 'ids of recordings that belong only to room with given room_friendly_id' do
```

Comment:
This sentence is confusing :laughing: 

`it 'returns recordings belonging to the room'`